setup.py

```diff
@@ -60,6 +60,12 @@
         "llvmlite <= 0.33.0;python_version>='3.6'",
         "llvmlite <= 0.31.0;python_version<'3.6'",
     ],
+    "opentelemetry": [
+        "opentelemetry-api == 0.9b0",
+        "opentelemetry-sdk == 0.9b0",
+        "opentelemetry-instrumentation == 0.9b0 ",
+        "opentelemetry-exporter-cloud-trace == 0.9b0",
```

Comment:
This package has been renamed to "opentelemetry-exporter-google-cloud"

Reply:
removed from nox since not needed and updated package name is referenced in documentation.